scipy/special/tests/test_basic.py

```diff
@@ -3265,7 +3265,7 @@ def test_legacy():
         assert_equal(special.nbdtr(1, 2, 0.3), special.nbdtr(1.8, 2.8, 0.3))
         assert_equal(special.nbdtri(1, 2, 0.3), special.nbdtri(1.8, 2.8, 0.3))
         assert_equal(special.pdtrc(1, 0.3), special.pdtrc(1.8, 0.3))
-        assert_equal(special.pdtr(1, 0.3), special.pdtr(1.8, 0.3))
+        assert_equal(special.pdtr(1.0, 0.3), special.pdtr(1.8, 0.3))
```

Comment:
Shouldn't need this test case anymore.

Reply:
sounds good I'll remove both `pdtr` and `pdtrc` in favor of separate pdtr test file